### ISSUE TITLE
Provide pre-calculated cost estimates for managed instance setups

### ISSUE THREAD

**Issue by caugustus-sourcegraph** [sourcegraph/sourcegraph#28422]
https://handbook.sourcegraph.com/product-engineering/engineering/enablement/delivery/managed/cost_estimation has a detailed process for calculating the cost of a managed instance which is hard to follow when looking for a ballpark number. We can improve this by pre-calculating some standard instance sizes and usage patterns.

Other improvements to consider:
- Providing sample historical data from existing instances
- Confirm with CE if there's a minimum estimate
- Update references to distribution

**Comment by kevinwojo** [comment]
Maybe this helps: https://docs.google.com/spreadsheets/d/1A-Tj427uL-x0MwXtkJzctOCmoKTQ_hGa06Rwp_7_FbE/edit?usp=sharing

I exported the CSV for projects with "sourcegraph-managed" and then created a pivot table.

https://console.cloud.google.com/billing/015B9F-927781-E3C63D/reports/tabular?organizationId=1006954638239&project=sourcegraph-managed-tinder

**Comment by JenRed777** [comment]
This work will probably be decided here:
https://docs.google.com/document/d/1aA6d2PY2fQA3IJr-6qL7T6sqQioBmumsxlz0T9ukDfg/edit#

**Comment by virginiaulrich** [comment]
Agreed we can close this one for now since we will have a lot more details as we think through path for MI to v2 and if this is still needed, will need to be based on that (but believe pricing has been finalized at this point)
